ios/FluentUI/Notification/NotificationTokens.swift

```diff
@@ -175,15 +175,15 @@ open class NotificationTokens: ControlTokens {
     /// The color of the outline around the frame of a notification
     open var outlineColor: DynamicColor {
         switch style {
-        case .primaryToast, .neutralToast, .primaryBar, .neutralBar, .dangerToast, .warningToast:
+        case .accentToast, .neutralToast, .accentBar, .neutralBar, .dangerToast, .warningToast:
             return DynamicColor(light: ColorValue.clear)
-        case .primaryOutlineBar:
-            return aliasTokens.strokeColors[.neutral1]
+        case .subtleBar:
+            return aliasTokens.strokeColors[.neutral2]
         }
     }
 
     /// The width of the outline around the frame of a notification
-    open var outlineWidth: CGFloat { GlobalTokens.borderSize(.thin) }
+    open var outlineWidth: CGFloat { 0.5 }
```

Comment:
why we not using border tokens?

Reply:
do we not have half the point?

Reply:
Looks like `.thin` is supposed to be 1 pixel on iOS, so it _should_ be right, but we have 1 set as the value, so we're actually returning 1 point. We should probably update our borderSize tokens to return something * screen pixel

Reply:
0.5 is not in our border tokens and the figma shows the border width as being 0.5:
<img width="1304" alt="image" src="https://user-images.githubusercontent.com/31874971/187747464-eacb1bcc-6387-49b7-9c98-921eadb870d3.png">

Reply:
seems like something we need to follow up with design?